frontend/src/scenes/ingestion/panels/VerificationPanel.tsx

```diff
@@ -1,21 +1,117 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { useActions, useValues } from 'kea'
 import { userLogic } from 'scenes/userLogic'
 import { useInterval } from 'lib/hooks/useInterval'
 import { CardContainer } from 'scenes/ingestion/CardContainer'
-import { Button, Row, Spin } from 'antd'
+import { Button, Row, Spin, Space, Popconfirm, Dropdown, Menu, Typography } from 'antd'
 import { ingestionLogic } from 'scenes/ingestion/ingestionLogic'
+import { DownOutlined, SlackSquareOutlined } from '@ant-design/icons'
+import { CreateInviteModalWithButton } from 'scenes/organization/Settings/CreateInviteModal'
+
+const { Text } = Typography
+import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
+import { FEATURE_FLAGS } from 'lib/constants'
 
 export function VerificationPanel(): JSX.Element {
     const { loadUser } = useActions(userLogic)
     const { user } = useValues(userLogic)
     const { setVerify, completeOnboarding } = useActions(ingestionLogic)
     const { index, totalSteps } = useValues(ingestionLogic)
+    const { featureFlags } = useValues(featureFlagLogic)
+    const [isMenuOpen, setIsMenuOpen] = useState(false)
 
     useInterval(() => {
-        !user?.team?.ingested_event && loadUser()
+        if (!user?.team?.ingested_event) {
```

Comment:
maybe you can simplify this to `if (!user?.team?.ingested_event && !isMenuOpen)`?

Reply:
done